src/clientSideScene/sceneEntities.ts

```diff
@@ -1375,13 +1360,135 @@ export class SceneEntities {
       groupOfDrafts.add(groupCircle)
     }
 
-    // The target of our dragging
-    let target: Object3D | undefined = undefined
+    const insertCircle3PointKclIntoAstSnapshot = (
+      points: Vector2[]
+    ): Program => {
+      if (err(kclCircle3Point) || kclCircle3Point.program === null)
+        return kclManager.ast
+      if (kclCircle3Point.program.body[0].type !== 'ExpressionStatement')
+        return kclManager.ast
+      if (
+        kclCircle3Point.program.body[0].expression.type !== 'CallExpressionKw'
+      )
+        return kclManager.ast
+
+      // The typechecking really made this ultra unergonomic. I wrote these to
+      // help out.
+      // x should be a LabeledArg but we've got none in wasm/lang?
```

Comment:
`LabeledArg` is defined in bindings. Maybe you need to add this export to the top of `wasm.ts`.

```typescript
export type { LabeledArg } from '../wasm-lib/kcl/bindings/LabeledArg'
```

...and then you should be able to import it here.